src/mainwindow.cpp

```diff
@@ -203,20 +205,16 @@ void MainWindow::load_config(QString filename) {
 
 		buildFilename();
 
-		// Check the wild-card-replaced filename to see if it exists already.
-		// If it does then increment the exp number.
-		// We only do this on settings-load because manual spin changes might indicate purposeful
-		// overwriting.
-		QString recFilename = QDir::cleanPath(ui->lineEdit_template->text());
-		// Spin Number
-		if (recFilename.contains(counterPlaceholder())) {
-			for (int i = 1; i < 1001; i++) {
-				ui->spin_counter->setValue(i);
-				if (!QFileInfo::exists(replaceFilename(recFilename))) break;
-			}
+		// Added by @Doug1983 to increase config file functionality:
```

Comment:
Authorship is normally preserved via git's author tags; otherwise the the comments are helpful